prExplorer/client-app/src/app/common/form/CustomTextInput.tsx

```diff
@@ -0,0 +1,22 @@
+import { useField } from 'formik';
+import React from 'react';
+import { Form, Label } from 'semantic-ui-react';
+
+interface CustomTextInputProps {
+    placeholder: string;
+    name: string;
+    label?: string;
+}
+
+export default function CustomTextInput(customTextInputProps: CustomTextInputProps) {
```

Comment:
Isn't this the same code you have already added in another PR? :P

Reply:
I changed propsName probally. :)